conf/messages.cy

```diff
@@ -596,10 +597,10 @@ settlorIndividualUkCountryOfResidencyYesNo.error.required = Dewiswch ‘Iawn’
 settlorIndividualUkCountryOfResidencyYesNo.heading = A yw {0} yn breswylydd yn y DU?
 settlorIndividualUkCountryOfResidencyYesNo.title = A yw’r setlwr yn breswylydd yn y DU?
 
-settlorIndividualUkCountryOfResidencyYesNoPastTense.title = Was the settlor a UK resident?
-settlorIndividualUkCountryOfResidencyYesNoPastTense.heading = Was {0} a UK resident?
-settlorIndividualUkCountryOfResidencyYesNoPastTense.checkYourAnswersLabel = Was {0} a UK resident?
-settlorIndividualUkCountryOfResidencyYesNoPastTense.error.required = Select yes if the settlor was a UK resident
+settlorIndividualUkCountryOfResidencyYesNoPastTense.title = A oedd y setlwr yn breswylydd yn y DU?
+settlorIndividualUkCountryOfResidencyYesNoPastTense.heading = A oedd {0} yn breswylydd yn y DU?
+settlorIndividualUkCountryOfResidencyYesNoPastTense.checkYourAnswersLabel = A oedd {0} yn breswylydd yn y DU?
+settlorIndividualUkCountryOfResidencyYesNoPastTense.error.required = Dewiswch ‘Iawn’ os yw’r setlwr yn breswylydd yn y DU // TODO: CHECK
```

Comment:
`settlorIndividualUkCountryOfResidencyYesNoPastTense.error.required` doesn't match the translation in the document